### PR TITLE
User blueprint

### PR DESCRIPTION
**Pull request by smotornyuk**

### CODE REVIEW

File: ckan/views/user.py [hunk 1]
```diff
@@ -0,0 +1,736 @@
...
+    is_sysadmin = authz.is_sysadmin(g.user)
+    try:
+        user_dict = logic.get_action(u'user_show')(context, data_dict)
+    except logic.NotFound:
+        h.flash_error(_(u'Not authorized to see this page'))
+        return
+    except logic.NotAuthorized:
+        base.abort(403, _(u'Not authorized to see this page'))
+
+    user_dict = user_dict
```
Comment (amercader) [inline ckan/views/user.py]: Redundant?

File: ckan/views/user.py [hunk 2]
```diff
@@ -0,0 +1,736 @@
...
+        page=page_number,
+        url=h.pager_url,
+        item_count=users_list.count(),
+        items_per_page=limit)
+
+    extra_vars = {u'page': page, u'q': q, u'order_by': order_by}
+    return base.render(u'user/list.html', extra_vars)
+
+
+# TODO: update after dashboard blueprint finished
```
Comment (amercader) [inline ckan/views/user.py]: Does this still need to be done?

File: ckan/views/user.py (hunk 1 above)
Comment (smotornyuk) [reply ckan/views/user.py]: Right

File: ckan/views/user.py (hunk 2 above)
Comment (smotornyuk) [reply ckan/views/user.py]: No, i forgot to remove this comment after adding dashboard routes.

**Comment by amercader** [comment]
:tada: :rocket:

**Pull request merged**